header.php

```diff
@@ -115,10 +115,10 @@
 
 						<?php
 
-						// Check whether the header search is deactivated in the customizer.
-						$disable_header_search = get_theme_mod( 'twentytwenty_disable_header_search', false );
+						// Check whether the header search is activated in the customizer.
+						$enable_header_search = get_theme_mod( 'twentytwenty_enable_header_search', false );
```

Comment:
This should be updated to have `true` as default value, to match the `add_control()`.

Reply:
Nice catch, @andersnoren! I've fixed this issue in https://github.com/WordPress/twentytwenty/pull/264